torch/_inductor/fx_passes/fuse_attention.py

```diff
@@ -723,6 +687,42 @@ def _sfdp_replacement_23(query, key, value):
     )
 
 
+def _sfdp_pattern_24(query, key, value, attention_mask):
```

Comment:
Is `pattern_24` just moved here for ordering, without any code change?

Reply:
Thanks for your comments. Yes. Just moved `pattern_24 `.